crates/http-sink/.hub/package-meta.yaml

```diff
@@ -1,6 +1,6 @@
 package_format_version: '0.3'
 name: http-sink
-version: 0.2.0
+version: 0.2.1
 group: infinyon
 description: HTTP sink connector reads records from data streaming and generates an HTTP request
 license: e.g. Apache2
```

Comment:
we should remove this file

Reply:
I was thinking we could merge this and apply this file removal on #38?